web/content/model/index.md

```diff
@@ -48,22 +48,20 @@ The partially defined theory functions may be defined using the following
 definitions in the model:
 
 ```smt2
-(define-fun div ((a Int) (b Int)) Int
+(refine-fun div ((a Int) (b Int)) Int
    (ite (= b 0) (ite (= a 0) 5 0) (div a b)))
-(define-fun car ((a (List Int))) (List Int)
+(refine-fun car ((a (List Int))) (List Int)
    (match a
       ((cons hd tl) hd)
       (nil  42)))
-(define-fun cdr ((a (List Int))) Int
+(refine-fun cdr ((a (List Int))) Int
```

Comment:
There is an old mistake here:  The return types of cdr and car are swapped.  cdr should return a (List Int) and car should return Int.